tests/test_examples.py

```diff
@@ -227,6 +228,20 @@ def test_c9104_module_name_violation() -> None:
     ), f"Failed {module_name_violation}. File does not add expected message."
 
 
+def _strip_output_format(input_filename: str) -> str:
+    with open(input_filename, "r") as file:
+        lines = file.readlines()
+
+    lines = [line for line in lines if not re.match(r".*output-format.*", line)]
+
+    with tempfile.NamedTemporaryFile(delete=False, mode="w", newline="") as temp_file:
```

Comment:
The overall idea here is good, but for tests it's better to use a pytest fixture (https://docs.pytest.org/en/stable/how-to/tmp_path.html); this will correctly ensure the file is cleaned up without the programmer needing to worry about it.